GameOfLife/Calculations/Rules.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    class Rules
+    {
+        public int[,] CellCalculation(int[,] GameField)
+        {
+            int[,] tempArr = (int[,])GameField.Clone();
+            int lenght = GameField.GetLength(0);
+            int width = GameField.GetLength(1);
+
+            for (int i = 0; i < lenght; i++)
+            {
+                for (int j = 0; j < width; j++)     // set copied same size array elements to 0
+                {
+                    tempArr[i, j] = 0;
```

Comment:
So, was there any point in cloning the GameField, if you are just resetting all the values in it anyway? Could just do:
int[,] tempArr = new int [lenght, width]; and you would get the same result (By default, all values in array will be set to 0, as that is the default value of int)

Reply:
Initialized as an array of default int values